lib/stack_master/parameter_resolvers/secret.rb

```diff
@@ -1,18 +1,23 @@
-require 'dotgpg'
+require 'os'
 
 module StackMaster
   module ParameterResolvers
     class Secret < Resolver
       SecretNotFound = Class.new(StandardError)
+      PlatformNotSupported = Class.new(StandardError)
 
-      array_resolver
+      unless OS.windows?
+        require 'dotgpg'
+        array_resolver  
+      end
 
       def initialize(config, stack_definition)
         @config = config
         @stack_definition = stack_definition
       end
 
       def resolve(value)
+        raise PlatformNotSupported, "Windows does not support the GPG Secret Parameter Resolver" if OS.windows?
```

Comment:
Maybe reword this to "The GPG Secret Parameter Resolver does not support Windows" ?